modules/apps/foundation/portal-configuration/portal-configuration-cluster/src/main/java/com/liferay/portal/configuration/cluster/internal/messaging/ConfigurationMessageListener.java

```diff
@@ -96,7 +91,7 @@ protected void reloadConfiguration(String pid, String filter, int type)
 					configuration.delete();
 				}
 				else {
-					configuration.update(properties);
+					configuration.update();
```

Comment:
@jonathanmccann were you able to test this particular change? I'm curious why the properties would come back null in the deleted line but not when calling `configuration.update`. Calling `configuration.update()` with no arguments ends up calling `persistenceManager.load(pid)` like the line you removed, but inside the framework.  Am I reading this wrong?  

If you tested it and it works then I'm fine with it, but at first glance it seems like it would be the same result in either case.